### ISSUE TITLE
Possible to run sim through random synthetic market data?

### ISSUE THREAD

**Issue by Primea1** [DeviaVir/zenbot#959]
I think it would be a good idea to append random synthetic market data to the end of a sim to see how it will react to sudden random market reactions. Is this possible?

**Comment by JensvdHeydt** [comment]
Yes, it is possible. You wouldn't even need to programm it. Simply add some random trades to the mongodb database for the corresponding timeframe.

**Comment by JensvdHeydt** [comment]
I’ve been thinking about this and it would actually be quite helpful for strategy testing and development if we had some fixed test data the bot could use. 
Maybe datasets for:
- Sideways market
- Rapidly rising/falling
- waving markets

**Comment by DeviaVir** [comment]
This issue was moved to highvelocityspace/zenbot#21
